docs/versioned_docs/version-2.11.0/troubleshooting/t-ndm.md

```diff
@@ -0,0 +1,100 @@
+---
+id: t-ndm
+title: Troubleshooting OpenEBS - NDM
+---
+
+## General guidelines for troubleshooting
+
+- Contact [OpenEBS Community](/docs/introduction/community) for support.
+- Search for similar issues added in this troubleshooting section.
+- Search for any reported issues on [StackOverflow under OpenEBS tag](https://stackoverflow.com/questions/tagged/openebs)
+
+[Blockdevices are not detected by NDM](#bd-not-detected)
+
+[Unable to claim blockdevices by NDM operator](#unable-to-claim-blockdevices)
+
+### Blockdevices are not detected by NDM {#bd-not-detected}
+
+One additional disk is connected to the node, with multiple partitions on the disk. Some of the partitions have a filesystem and is mounted. `kubectl get bd -n openebs` does not show any blockdevices. Ideally the blockdevice resources for the partitions should have been shown.
+
+```shell hideCopy
+NAME   FSTYPE MOUNTPOINT   SIZE
+sda                        1.8T
+├─sda1                     500G
+├─sda2                     500G
+├─sda3                     500G
+└─sda4 ext4   /kubernetes  363G
+sdb                       55.9G
+├─sdb1 vfat   /boot/efi    512M
+└─sdb2 ext4   /           55.4G
+```
+
+**Troubleshooting:**
+
+Check the output of `lsblk` on the node and check the mountpoints of the partitions. By default NDM excludes partitions mounted at `/, /boot` and `/etc/hosts` (which is same as the partition at which kubernetes / docker filesystem exists) and the parent disks of those partitions. In the above example `/dev/sdb` is excluded because of root partitions on that disk. `/dev/sda4` contains the docker filesystem, and hence `/dev/sda` is also excluded.
+
+**Resolution:**
+
+The `ndm-config-map` needs to be edited.
+
+1. Remove `/etc/hosts` entry from the os-disk-exclude-filter
+2. Add the corresponding docker filesystem partition in exclude section of path filter. eg: `/dev/sda4`
+3. Restart the NDM daemonset pods.
+
+The blockdevices should now be created for the unused partitions.
+
+### Unable to claim blockdevices by NDM operator{#unable-to-claim-blockdevices}
+
+BlockDeviceClaims may remain in pending state, even if blockdevices are available in Unclaimed and Active state. The main reason for this will be there are no blockdevices that match the criteria specified in the BlockDeviceClaim. Sometimes, even if the criteria matches the blockdevice may be in an Unclaimed state.
+
+**Troubleshooting:**
+
+Check if the blockdevice is having any of the following annotations:
+
+1.
+
+```
+metadata:
+  annotations:
+    internal.openebs.io/partition-uuid: <uuid>
+    internal.openebs.io/uuid-scheme: legacy
+```
```

Comment:
for the list items please add some space, so we can wrap the children inside the li

Reply:
kind of bad UX due to (no content before code)
<img width="757" alt="Screenshot 2021-08-03 at 12 20 02 AM" src="https://user-images.githubusercontent.com/6524419/127909511-f58b5143-8988-4914-9dbf-ccaa2c36afa0.png">